### PR TITLE
Mailmap and RELEASE.md updates in preparation for 0.8

### PR DESCRIPTION
**Pull request by Cadair**

### CODE REVIEW

File: .mailmap [hunk 1]
```diff
@@ -13,9 +13,11 @@ Stuart Mumford <stuart@cadair.com> Stuart Mumford <s.mumford@sheffield.ac.uk>
...
 Stuart Mumford <stuart@cadair.com> Bertie Smith <stuart@cadair.com>
 Florian Mayer <florian.mayer@bitsrc.org> Florian Mayer <flormayer@aim.com>
-David Perez-Suarez <dps.helio@gmail.com> dpshelio <dps.helio@gmail.com>
-David Perez-Suarez <dps.helio@gmail.com> DavidPS <dps.helio@gmail.com>
-David Perez-Suarez <dps.helio@gmail.com> David PS <dps.helio@gmail.com>
+David Pérez-Suárez <dps.helio@gmail.com> dpshelio <dps.helio@gmail.com>
+David Pérez-Suárez <dps.helio@gmail.com> DavidPS <dps.helio@gmail.com>
+David Pérez-Suárez <dps.helio@gmail.com> David PS <dps.helio@gmail.com>
+David Pérez-Suárez <dps.helio@gmail.com> David Perez-Suarez <dps.helio@gmail.com>
+David Pérez-Suárez <dps.helio@gmail.com> David Perez-Suarez <d.perez-suarez@ucl.ac.uk>
```
Comment (dpshelio) [inline .mailmap]: I approve this! 👍

File: RELEASE.md [hunk 2]
```diff
@@ -1,48 +1,45 @@
...
     Stuart Mumford
-    Pritish Chakraborty
-    Steven Christe
     Jack Ireland
+    Punyaslok Pattnaik
+    Nabil Freij
+    Alex Hamilton
+    Ankit Kumar *
+    Steven Christe
+    David Pérez-Suárez
```
Comment (dpshelio) [inline RELEASE.md]: That will change once #2253 is merged.. right?

Comment (nabobalis) [inline RELEASE.md]: Will this be the only change?

**Comment by nabobalis** [comment]
Merge 4/5

**Pull request merged**